src/com/cloudogu/ces/cesbuildlib/Maven.groovy

```diff
@@ -113,7 +114,7 @@ abstract class Maven implements Serializable {
         useRepositoryCredentials(config)
     }
 
-    void useMirrors(Map<String, String>... configs) {
```

Comment:
By using a typed Map, you will get a "Possible heap pollution via varargs parameter" Warning instead. Because the use of this methodes from outside this context is not type safe. The methods itself use a custom Object as value not a String. (Line 120 in this File)